core/src/components/select/select.tsx

```diff
@@ -471,10 +471,10 @@ export class Select implements ComponentInterface {
           'select-disabled': disabled,
         }}
       >
-        <div class={selectTextClasses}>
+        <div class={selectTextClasses} part="select-text">
```

Comment:
Do we want parts to be prefixed with select? In Segment we just add `part="indicator"` since the way you will target would be like:

```
ion-segment::part(indicator)
```

Reply:
👍  I removed the prefix